package.json

```diff
@@ -91,13 +91,18 @@
   },
   "dependencies": {
     "@tanstack/react-query": "^4.28.0",
+    "classnames": "^2.3.2",
+    "dayjs": "^1.11.7",
     "seamapi": "^5.19.0",
     "uuid": "^9.0.0"
   },
   "devDependencies": {
     "@babel/preset-env": "^7.21.4",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.21.4",
+    "@emotion/styled": "^11.10.6",
+    "@mui/icons-material": "^5.11.16",
+    "@mui/material": "^5.12.2",
```

Comment:
fine for now based on usage

Reply:
These are only to quickly scaffold out examples in the stories. Won't be using them in the actual components, and they won't be included in the consumer's final bundle.